libs/agno/agno/agent/agent.py

```diff
@@ -4277,12 +4277,18 @@ def get_system_message(self, session_id: str, user_id: Optional[str] = None) ->
         if self.instructions is not None:
             _instructions = self.instructions
             if callable(self.instructions):
-                _instructions = self.instructions(agent=self)
+                import inspect
```

Comment:
i think same update needed for `Team`.. 
LGTM

Reply:
Done!